Extensions/keyboardShortcut.js

```diff
@@ -109,7 +109,7 @@
     }
 
     function clickQueueButton() {
-        document.querySelector(".control-button-wrapper .spoticon-queue-16").click();
+        document.querySelector(".main-nowPlayingBar-right > div.ExtraControls > div.GlueDropTarget > div.control-button-wrapper > button").click();
```

Comment:
Let's keep it simple to reduce our dependency on the CSS Mappings: `div.ExtraControls div.control-button-wrapper > button`